.github/CONTRIBUTING.md

```diff
@@ -21,6 +21,12 @@ If you are reading this page, you are possibly interested in contributing to our
 
 Please see our [wiki article](https://ardupilot.org/dev/docs/submitting-patches-back-to-master.html).
 
+# Addressing ArduPilot github Issues
+
+We typically do not assign issues - *particularly* "Good First Issue"s to specific contributors.
```

Comment:
Thanks for this.  I'd go with positive comment first.  So provide a link to our issues list suggesting they look through the list and find something they're interested in working on.  Suggest items labelled with "good first issue" (and maybe even provide a link that automatically selects that label... then follow-up with the comment about how we don't assign issues

Reply:
I've added some more words (and a link...)